docs/blocks-storage/compactor.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Compactor"
```

Comment:
We should mention that Compactor would need a decently sized disk, maybe even provision some PVCs if disk on node is not enough?

Reply:
Very good point. I totally missed it. I'm going to add it both to the compactor doc and production tips.